### PR TITLE
Mapbox GL - Add rotation/pitch alignment methods to Marker

### PR DESCRIPTION
**Pull request by webberig**
As requested per https://github.com/Wykks/ngx-mapbox-gl/pull/226

### CODE REVIEW

**Review by dmytro-gokun — COMMENTED** [review]
Please add some tests (using $ExpectType)

**Review by dmytro-gokun — CHANGES_REQUESTED** [review]
Please add some tests (using $ExpectType)

**Comment by webberig** [comment]
Just out of curiosity...
```
$ npm run test
Error: 'git diff' should be empty. Following files changed:
types/mapbox-gl/mapbox-gl-tests.ts
```

How do I actually run the tests without having to commit potential garbage :-)

File: types/mapbox-gl/mapbox-gl-tests.ts [hunk 1]
```diff
@@ -599,8 +599,12 @@ let marker = new mapboxgl.Marker(undefined, {
     pitchAlignment: 'viewport',
 })
     .setLngLat([-50, 50])
+    .setPitchAlignment("map")
+    .setRotationAlignment("viewport")
     .addTo(map);
 
+expectType<mapboxgl.Alignment>(marker.getPitchAlignment());
```
Comment (dmytro-gokun) [inline types/mapbox-gl/mapbox-gl-tests.ts:606]: Please use $ExpectType for new code.

**Review by dmytro-gokun — CHANGES_REQUESTED** [review]

**Comment by dmytro-gokun** [comment]
> Just out of curiosity...

https://github.com/DefinitelyTyped/DefinitelyTyped#verifying

**Comment by webberig** [comment]
Oh ok... That's a bit confusing because `lint` and `test` are usually different...

**Review by dmytro-gokun — APPROVED** [review]

**Comment by webberig** [comment]
Ready to merge

**Pull request merged**